data/hai/hai missions.txt

```diff
@@ -148,12 +148,12 @@ mission "Assisting Hai"
 	to offer
 		random < 27
 	source
-		government "Hai"
+		government "Hai" "Hai Merchant" "Hai Merchant (Sympathizers)" "Hai Merchant (Human)"
```

Comment:
```suggestion
		government "Hai" "Hai Merchant" "Hai Merchant (Human)"
```
there is nothing in hai space that could disable the sympathizers since they arent in war with the uhai, and implying that they care less about you repping them could be appropriate?